modules/internal/String.chpl

```diff
@@ -807,11 +807,19 @@ module String {
       compilerError("substring removed: use string[range]");
     }
 
+    /*
+     Deprecated, use :proc:`string.isEmpty`.
+     */
+    inline proc isEmptyString() : bool {
+      compilerWarning("'isEmptyString' has been deprecated, use 'isEmpty' method instead");
```

Comment:
I'd go with something like `compilerWarning("isEmptyString is deprecated - please use isEmpty instead")` to be more consistent with other deprecation warnings (which also makes me think we should have a deprecatedWarning wrapper)

Reply:
Agreed. Want to open an issue on that or shall I?

Reply:
Go for it

Reply:
Done: https://github.com/chapel-lang/chapel/issues/12438